package.json

```diff
@@ -8,12 +8,17 @@
         "db:pack": "node tooling/pack-macros.mjs",
         "new-effect": "node tooling/create-new-effect.mjs",
         "test": "npm run test:js",
-        "test:js": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
+        "test:js": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+        "format": "prettier --write scripts packs_source lang tooling test",
+        "prepare": "npx husky init"
```

Comment:
`"prepare"` sets up `husky` whenever a bare `npm install` is run. See: https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-scripts